metriq_gym/process.py

```diff
@@ -43,12 +43,15 @@ def poll_job_results(jobs_file: str) -> list[BenchJobResult]:
             result = BenchJobResult(**(json.loads(line)))
             job = get_job(result)
             status = job.status()
-            if status == JobStatus.RUNNING:
+            if (status == JobStatus.RUNNING) or (result.job_type != job_type):
```

Comment:
[nit]: I think more modern Python prefers using `is` and `is not`, respectively in these types of lines for `==` and `!=`. Also, we can drop the parens around these objects as they are redundant.

Reply:
I disagree. This is true for `None` or when used with key word `in`, but not value-type comparisons.